### PR TITLE
Fix #849: Show that your guide is still a draft

### LINKED ISSUE #849: Show that your guide is still a draft
In the list of guides on My Profile I see a bunch of guides, but now that we have draft functionality it's not clear which ones of those are drafts:

screenshot 2016-10-09 10 04 58

Expected:

See which guides are drafts.

How I would implement this:

1. Decide - design-wise - how I would indicate that something is a draft.
2. In the .html.erb I would use the `.draft` indicator on a guide to differentiate that.
3. Do a PR. 
4. Enjoy a beverage of choice.

### PR DESCRIPTION
**Pull request by jomarquez**

### CODE REVIEW

File: app/assets/stylesheets/styles/_presets.scss [hunk 1]
```diff
@@ -47,6 +47,16 @@
   border: darken($gray-bg, 10%);
 }
 
+@mixin label-variant($color){
+  background-color: $color;
+  &[href] {
```
Comment (simonv3) [inline app/assets/stylesheets/styles/_presets.scss]: Just add a newline between lines 51 and 52, so that `&[href]` is on line 53.

**Comment by simonv3** [comment]
Thanks for the PR and fixes @jomarquez, and for baring with Hound :)

File: app/assets/stylesheets/styles/_presets.scss (hunk 1 above)
Comment (jomarquez) [inline app/assets/stylesheets/styles/_presets.scss]: ahh!, thanks so much @simonv3

**Comment by simonv3** [comment]
Woop, this will deploy on [staging](http://staging.openfarm.cc) and you'll be able to see and test it there - I'll probably do a deploy from staging to production later today :)

**Pull request merged**